etc/cc-localhost.conf

```diff
@@ -2,16 +2,25 @@
 # Warning: This configuration is insecure, users can hijack each other's perms.
 #
 # The easiest way to use this file is to leave it unedited and instead
-# start nginx using bin/nginx-dev, which will dynamically creat# suitable config files on the fly.
+# start nginx using bin/nginx-dev, which will dynamically create
+# suitable config files on the fly.
+
+# Configuration applying to all servers.
+error_page 502 503 504 =503 /static/503.html;
+
+proxy_set_header Host $http_host;
+proxy_next_upstream_tries 1;
```

Comment:
Turns out this causes (or at least surfaces) an unanticipated problem; see #416.